src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,238 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int NOT_FOUND = -1;
+    private static final int ONE = 1;
+    private static final int HALF = 2;
```

Comment:
sense of extracting numbers to the constants is to give more explanations about them by defining constant name, but your name explains nothing, so it makes no sense
```suggestion
```